notebooks/example/anomaly_code.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<p>How do we interpret these outputs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/HTM-applications/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>